gammapy/scripts/tests/test_image_pipe.py

```diff
@@ -64,9 +64,8 @@ def test_image_pipe(tmpdir):
     mosaic = MosaicImage(image, energy_band=energy_band, offset_band=offset_band, data_store=data_store,
                          obs_table=data_store.obs_table, exclusion_mask=exclusion_mask)
     mosaic.make_images(make_background_image=True, for_integral_flux=True, radius=10.)
-    # TODO: this fails on some machines, for unknown reasons.
-    #assert_equal(mosaic.maps['counts'].data.sum(), 2334.0)
-    #assert_almost_equal(mosaic.maps['bkg'].data.sum(), 1987.2088283446637)
-    #assert_almost_equal(mosaic.maps['exposure'].data.sum(), 5.9292301086257267e17)
-    #assert_almost_equal(mosaic.maps['significance'].lookup(center), 33.699068269123039)
-    #assert_almost_equal(mosaic.maps['excess'].data.sum(), 346.79117165533626)
+    assert_allclose(mosaic.maps['counts'].data.sum(), 2334.0, atol=3)
```

Comment:
> Right? Maybe rounding to a reasonable precision before binning events into a sky map could solve the problem?

That doesn't sound right. Higher precision means smaller "edge area" around the sky map where floating point jitter can move events inside and outside.

Feel free to reduce the tolerance here ... probably with 0 you'll get back the issue and with 1 it will work.
(The atol on the other asserts can be removed.)

Debugging this via travis-ci is very painful and possibly pointless.
Histograms are always unstable wrt. which bin is hit this way.

One option might be to locally compute the distance of the events to the edge of the image and find the smallest distance. If it's without 32 bit or 64 bit float precision, then that's the event that's sometimes in and sometimes out.
Again ... I'm not sure it's worth your time ...